modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/construction/ConstructionService.scala

```diff
@@ -120,8 +121,9 @@ object ConstructionService {
           }.flatMap { proof =>
             EitherT
               .fromEither(
-                KryoSerializer[F]
+                JsonBinarySerializer
                   .serialize(Signed[DAGTransaction](transaction, NonEmptySet.of(proof)))
+                  .asRight[Throwable]
```

Comment:
Shouldn't this be `ConstructionError` rather than `Throwable` ?

Reply:
Changed.